### ISSUE TITLE
RFE: Outdated documentation for "Extending the basic network layout" 

### ISSUE THREAD

**Issue by gnyers** [Kubeinit/kubeinit#391]
**Describe the bug**
The following issues when trying to follow the instructions for ["Extending the basic network layout"](https://www.anstack.com/blog/2020/08/25/KubeInit-External-access-for-OpenShift-OKD-deployments-with-Libvirt.html#extending-the-basic-network-layout) in the document: 

- the Ansible variables mentioned [here](https://www.anstack.com/blog/2020/08/25/KubeInit-External-access-for-OpenShift-OKD-deployments-with-Libvirt.html#step-2-adjusting-the-deployment-command) are nowhere to be found: 
  - ``kubeinit_libvirt_external_service_interface_enabled``, 
  - ``kubeinit_libvirt_external_service_interface``
  - ``kubeinit_inventory_network_bridge_external``,

- would be nice to include this description into repo

**To Reproduce**
1. Clone repo
2. run cmd ``grep -irHn ANY_VARNAME_ABOVE``; no hit.

**Comment by ccamacho** [comment]
Hello @gnyers, you are right documentation for the main branch is currently out of date, we introduced some major changes that need to be documented.

What you can do is to use the latest release available which should have those variables.

I will leave this open so when we fix the docs we can cut a new release.

Please feel free to ask any questions you have, and if you find any issue also feel free to raise issues in the project's repo (https://www.github.com/kubeinit/kubeinit). You can jump in and ask in the slack channel https://kubernetes.slack.com/archives/C01FKK19T0B Also, it would be awesome if you can star the project to catch up with updates and new features.

**Comment by gnyers** [comment]
Hi @ccamacho; I've pulled the latest version less than 5 min. ago, but still no sign of these variables. What commit are you referring to? Or have they been changed? Anyway, I'm not sure how to proceed. 

With regards to the slack channel; no luck there either: can not join with my existing account. 

I'd be happy to test the procedure, but there should be instructions to follow.

**Comment by ccamacho** [comment]
This https://github.com/Kubeinit/kubeinit/releases/tag/1.0.2

https://docs.kubeinit.com/usage.html#running-from-a-release

**Comment by drejmar** [comment]
I have a dedicated server in cloud (contabo).
They provide one IPv4 address (/24). Based on your documentation I ordered one additional IP to be assigned to the service VM.
This new IP has range /19, meaning that I need to specify gateway and netmask.

I was able to assign it to:
the server,
a docker container,
a LXD container.

I cannot assign it to the service VM.
Problems:
it's not accessible from Internet,
after making it default route it is accessible, but then other VMs cannot communicate with each other,
after assigning it to the server and using NAT (iptables) it is accessible, but it doesn't work, because web sockets connections fail.

The only way I'm able to use the installation is to use OpenVPN server, exposing internal networks to the client.

I wish you will soon find a solution which can expose OKD installation to the internet
